sound/hybrid_font.h

```diff
@@ -799,8 +803,44 @@ class HybridFont : public SaberBase, public Looper {
 	SaberBase::DoEffect(EFFECT_POSTOFF, saved_location_);
       }
     }
+    // Wait for retraction sounds to finish, then play idle.wav
+#ifdef ENABLE_IDLE_SOUND
+    if (idling_) {
+      if (GetWavPlayerPlaying(&SFX_in) || GetWavPlayerPlaying(&SFX_pstoff)) {
+        return;
+      } else {
+        idling_ = false;
+        StartIdleSound();
```

Comment:
Maybe we should fade the idle sound in?
Or play it gapless after the retraction sound?
Or start it before "in" is done to allow some overlap?

Reply:
I love the idea.
Fade in works great for me.
Updated.